src/modules/common/components/SearchInput.js

```diff
@@ -11,19 +10,21 @@ export class SearchInput extends React.Component {
     filterText: PropTypes.string.isRequired,
     onFilterTextChange: PropTypes.func.isRequired,
     spaceSearch: PropTypes.bool,
+    onClick: PropTypes.func,
```

Comment:
onClick makes you think it's the onClick for the whole componet. Maybe onClickInput would be better.

Reply:
Done